### ISSUE TITLE
[front/back] Redesign Homepage

### ISSUE THREAD

**Issue by biliboss** [OOTOPIA/ootopia-app#162]
### Background
We will apply the new layout and new interactions for the home screen of the Ootopia application - in it the application of the new Design System already consists. This demand consists of a redesign of the homepage and although it has some functionality interactions the backend functionalities will be realized in another demand.

### Jira Card
https://devmagic.atlassian.net/browse/OOT-146

### What should be done
New layout of the interactions - in the gif below we have all the interactions of the new system timeline.
![Mucung6Iay.gif](https://images.zenhubusercontent.com/60d9a19222b134d7fe8709b8/ec469a58-ebac-468d-8c7a-9057bf290278)

### Upper area - Menus

**Clock icon in the upper area**
Make interaction where the user can show/hide the remaining time - this can be a fixed data, the demand responsible for correctly configuring the game's end time in another demand.

**Left Menu**
Currently this menu is hidden inside the user's profile, we will bring it to the front and after redesigning some interactions we will add the options. For now we will keep only the options:
* Profile; If the user is logged out it leads to the login flow, if logged in it leads to the user page;
* Logout: Only appears if logged in.

### Game Icons
The three circles on the left side of the Ooz represent the gamefication, it is equivalent to the 3 little bars on the top area of the application that exist today (image below). Each one represents different visual representations of the game levels. From left to right

![image.png](https://images.zenhubusercontent.com/60d9a19222b134d7fe8709b8/1535cf5e-5899-4e67-a7c4-45058f0be5d2)

* Global  - _Not present in the MVP, will be entered as V1_
* City - _Not present in the MVP, will be entered as V1_
* Personal - _When the user clicks an animation should happen showing the time that the user is in the application, the time that he configured and the percentage of that time. The circle should be filled according to the percentage of the circle.In this demand the focus will be to complement the demand https://app.zenhub.com/workspaces/ootopia-60b82d5c8990d7000f175d3f/issues/ootopia/ootopia-app/162 and apply the business rules (backend integration) that notifies the user how much time was used / how much time was scheduled._

### Game Circles - City and Global  Additional Information
_These two circles won't have interactions concerning the game, but if the user touches them a brief explanation should happen - this explanation should not be done in this quest, it will be done in the quest, we are waiting for a feedback from the client (image below) @marcelobohrerid _
![image.png](https://images.zenhubusercontent.com/60d9a19222b134d7fe8709b8/65e0bef9-6872-4784-a873-4968f438e5d4)

### Bottom Menu and Interactions

**Bottom Menu**
A new menu was made with the icons:
* **Home**: Main Page;
* **Bussola**: Learning Tracks (still to be done);
* **More**: Takes the user to the photo/video stream;
* **OO**: Takes the user to the portfolio (still to be done)
* **Profile**: Takes the user to the user's screen

### Bottom Menu - Additional Information
All the icons used in the redesign are part of the Feather Icons gallery, but the ones below (bottom menu) had to go through a layout adaptation - originally they were just lines, but a weight (solid color) was added to some elements. I took the liberty of making the adjustments and making these icons available for download.

### Gamefication Points

The "**accumulated**" points of the day are shown on the homepage with the "OOz" area - the actions that generate OOz appear in an animation, example: "+20.0" → adds to accumulated → OOz equals 20.00.

The actions that generate this are described in the gamefication document, some examples: Access and consume learning tracks, post videos, post photos, gratitudes, etc.

In the gif below we have an example of how the points are shown.
![P0RpoxwqPt.gif](https://images.zenhubusercontent.com/60d9a19222b134d7fe8709b8/61add159-f0e4-40eb-8858-064b63e0e10d)

### Videos
It was agreed with @marcelobohrerid that the videos will be squared on the timeline, when the user uploads a vertical or horizontal video and it doesn't fit in this square area it's interesting to add black borders on the sides. @claudiooliveira @AndrewDiass  Will this be possible? if there is a need for alignment just call .

### Learning Tracks
Although the XD reference has Learning Tracks, it will not be present in this quest. There is an epic just to deal with its visuals and interactions  

### XD Links
 https://xd.adobe.com/view/d48e6488-b821-4a39-a4f0-63bc0cd781b2-81dd/

**Comment by claudiooliveira** [comment]
**About the daily goal of gamification:**

We have internally decided that there will be a global timetable that will affect all users and their daily goals. Then the cycle starts at 00:00:00 UTC and ends at 23:59:59 UTC.

Following the time established above, some examples of the start and end of the daily goal for each country:

Brazil - Start: 21:00: - End: 20:59:59
England - Start: 01:00: - End: 00:59:59

In short, while the daily cycle to meet the established goal for a Brazilian citizen is 9 pm, that of an English citizen is 1 am.

The global daily cycle will end at the same time for everyone regardless of their time zone.
